test/work_item_repository.go

```diff
@@ -22,32 +22,34 @@ type WorkItemRepository struct {
 		result1 *app.WorkItem
```

Comment:
Wonder if it's time to kill this stub thinggy... Seems like everyone is patching it due to API changes, but none using it?